app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -35,12 +35,13 @@ class ReturnsJourneyNavigator {
 
     exported >= (manufactured + imported)
   }
-
+  
+  @deprecated("Call direct route method on this class instead", since = "19th July 2022")
   val normalRoutes: PartialFunction[Page, UserAnswers => Call] = {
-
+    // TODO - replace with direct calls
```

Comment:
Is this TODO in this PR or do we need a ticket to come back to this?

Reply:
// TODO fix all the TODOs

:smiling_imp: 

More of a wishful to-do I think than something for right now